README.md

```diff
@@ -15,3 +15,13 @@ Support for x86_64 specific instructions (e.g. TLB flush), registers (e.g. contr
 
 This needs to have the [compile-time requirements](https://github.com/alexcrichton/cc-rs#compile-time-requirements) of the `cc` crate installed on your system.
 It was currently only tested on Linux and MacOS.
+
+## Enforcing external_asm build
+
+In addition to the standard cargo build the `external_asm` build path needs to
+be exercised:
+
+```
+cargo build --no-default-features --features external_asm,instructions
+cargo test --no-default-features --features external_asm,instructions -- --test-threads=1
+```
```

Comment:
The README is only targeted at the crate's users not the developers, so perhaps this could be moved to a CONTRIBUTING file. We could also mention that this is only necessary when adding/modifying instruction functions.

Reply:
I can do that.